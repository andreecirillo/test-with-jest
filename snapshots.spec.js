const user = {
    name: 'Tony Tinkerton',
    age: 42,
    job: 'inventor'
}

test('user matches', () => {
    expect(user).toMatchSnapshot()
})