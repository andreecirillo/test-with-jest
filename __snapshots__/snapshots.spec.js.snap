// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user matches 1`] = `
{
  "age": 42,
  "job": "inventor",
  "name": "Tony Tinkerton",
}
`;
